### DISCUSSION TITLE
[Layout children]: No route named "search" exists in nested

### DISCUSSION THREAD

**Discussion by JoshuaAmaju** [expo/router#222] [Q&A]
Getting a `LogBox.js:154 [Layout children]: No route named "search" exists in nested` warning when using a nested layout folder structure as shown below.

```
.
└── app/
    ├── (root)/
    │   ├── index.tsx
    │   ├── search/
    │   │   └── index.tsx
    │   ├── profile/
    │   │   └── index.tsx
    │   └── _layout.tsx
    ├── create/
    │   └── index.tsx
    └── _layout.tsx
```

Any idea what's wrong

**Comment by ThaJay** [comment]
This warning is bs because the folder `my-sensors` exists and it contains an `index.ts` and it renders with no error or warning when I visit the route in the app.

Im my case it seems to come from: `useScreens.tsx:53:16` `project_folder/node_modules/expo-router/src/useScreens.tsx`
```
[Layout children]: No route named "my-sensors" exists in nested children: 
Array(7) [ "[...missing]", "index", "live-view", "my-sensors/[sensor-id]", "my-sensors/index", "my-sensors/kabasd_layout", "_sitemap" ]
​
0: "[...missing]"
​
1: "index"
​
2: "live-view"
​
3: "my-sensors/[sensor-id]"
​
4: "my-sensors/index"
​
5: "my-sensors/kabasd_layout"
​
6: "_sitemap"
​
length: 7
```

(kabasd_layout is there because it was `_layout` before but then it gives the same warning but for `"my-sensors/[sensor-id]"`.)
